composer.json

```diff
@@ -9,13 +9,14 @@
         "nyholm/psr7": "^1.3",
         "sensio/framework-extra-bundle": "^5.1",
         "symfony/console": "^5.1",
+        "symfony/dotenv": "^5.1",
         "symfony/flex": "^1.8",
         "symfony/framework-bundle": "^5.1",
         "symfony/http-client": "^5.1",
         "symfony/monolog-bundle": "~3.5",
+        "symfony/security-core": "5.1.*",
```

Comment:
Here is the only "real" change. We were using classes from this package without having it installed